src/middlewares/authorization-verifier.js

```diff
@@ -11,7 +11,7 @@ module.exports = function createAuthorizationVerifier(userRepository) {
 
   // eslint-disable-next-line complexity
   function extractAuthorizationToken(request, response, next) {
-    const token = request.headers.token || request.json.token || request.body.token;
```

Comment:
Why did you removed this line ?

Reply:
We changed it to extract the token only from 'request.headers'.